proto/poktroll/proof/params.proto

```diff
@@ -12,6 +12,7 @@ message Params {
   option (amino.name) = "poktroll/x/proof/Params";
   option (gogoproto.equal) = true;
 
+  // TODO_FOLLOWUP(@olshansk, #690): Either delete this or change it to be named "minimum"
```

Comment:
Could you explain why this can be deleted?

Reply:
I see now. Could this be a map of serviceId to difficulty target hash (string->bytes)?

Reply:
That's what the `RelayMiningDifficulty` on-chain type already is. See `/Users/olshansky/workspace/pocket/poktroll3/x/tokenomics/keeper/update_relay_mining_difficulty.go`.

```go
type RelayMiningDifficulty struct {
	ServiceId string `protobuf:"bytes,1,opt,name=service_id,json=serviceId,proto3" json:"service_id,omitempty"`
	BlockHeight int64 `protobuf:"varint,2,opt,name=block_height,json=blockHeight,proto3" json:"block_height,omitempty"`
	NumRelaysEma uint64 `protobuf:"varint,3,opt,name=num_relays_ema,json=numRelaysEma,proto3" json:"num_relays_ema,omitempty"`
	TargetHash []byte `protobuf:"bytes,4,opt,name=target_hash,json=targetHash,proto3" json:"target_hash,omitempty"`
}
```

The baseline is just `fffff`. 

Having an on-chain parameter for just extra overhead and context.